engine/frontend/src/api/enclave.js

```diff
@@ -19,6 +19,13 @@ export const getEnclavesFromKurtosis = async (token, apiHost) => {
     return []
 }
 
+export const removeEnclave = async (token, apiHost, enclaveName) => {
+    const response = await removeEnclaveFromEnclaveManager(enclaveName, token, apiHost)
+    const enclave = response.enclaveInfo;
```

Comment:
what does the `enclave` variable do?

Reply:
just prints undefined, was using this for testing, forgot to remove it.